pkg/api/admin/openshiftcluster_convert.go

```diff
@@ -129,6 +130,7 @@ func (c *openShiftClusterConverter) ToInternal(_oc interface{}, out *api.OpenShi
 		}
 	}
 	out.Properties.ProvisioningState = api.ProvisioningState(oc.Properties.ProvisioningState)
+	out.Properties.ProvisionedBy = oc.Properties.ProvisionedBy
```

Comment:
please add a test to pkg/api/admin/openshiftcluster_validatestatic_test.go to validate that this field is immutable in the admin API

Reply:
done